includes/options/class-amp-options-menu.php

```diff
@@ -232,6 +243,52 @@ public function render_theme_support() {
 		<?php
 	}
 
+	/**
+	 * Render mobile redirect setting.
+	 *
+	 * @since 1.6
+	 */
+	public function render_mobile_redirect() {
+		$can_customize_mobile_settings = apply_filters( 'amp_customizer_is_enabled', true );
+		?>
+		<p>
+			<label for="mobile_redirect">
+				<input id="mobile_redirect" type="checkbox" name="<?php echo esc_attr( AMP_Options_Manager::OPTION_NAME . '[mobile_redirect]' ); ?>" <?php checked( AMP_Options_Manager::get_option( Option::MOBILE_REDIRECT ) ); ?>>
+				<?php esc_html_e( 'Redirect mobile visitors to the AMP version of a page.', 'amp' ); ?>
+			</label>
+		</p>
+		<?php
+		if ( $can_customize_mobile_settings ) :
+			$mobile_customizer_url = add_query_arg(
+				[
+					'autofocus[section]' => 'amp_mobile',
```

Comment:
This is tricky because in the new Reader mode when a theme is selected, this section will not exist. I wonder actually if we shouldn't just omit the setting altogether for now, but allow it to disabled via filter.

Reply:
In other words, this PR could eliminate the Exit Reader Mode link, the toggle for this setting, and any toggle for showing the “Leave mobile site” link in the footer. Should this be a decision we make, as opposed to providing a option?

Reply:
The "leave mobile site" link can be displayed unconditionally in the footer of the AMP version. Then in the non-AMP version the footer link can be displayed conditionally based on whether the user has mobile redirection enabled. This would make it simpler. (I realize this deviates from the requirements I laid out, but after thinking about it more I think this would be a good change.)

Reply:
> This is tricky because in the new Reader mode when a theme is selected, this section will not exist. I wonder actually if we shouldn't just omit the setting altogether for now, but allow it to disabled via filter.

Disabling via a filter would make the most sense here then.

---

The new Reader mode will effectively render the current AMP Customizer panel useless, correct? If so, then I would suggest removing it now then.

Reply:
> The "leave mobile site" link can be displayed unconditionally in the footer of the AMP version. Then in the non-AMP version the footer link can be displayed conditionally based on whether the user has mobile redirection enabled.

OK sounds good. We can filter showing the exit link and the text as well then.

Reply:
> The new Reader mode will effectively render the current AMP Customizer panel useless, correct? If so, then I would suggest removing it now then.

This is a bit tricky. The Reader mode will effectively render the AMP Customizer panel useless _if_ a Reader theme is selected. However, it is still needed if the Classic theme is selected.

> The new Reader mode will effectively render the current AMP Customizer panel useless, correct? If so, then I would suggest removing it now then.

The exit link in the header? I propose actually that we remove it entirely in favor of only having the link in the footer. The header exit link is problematic for other reasons, e.g. #4573.

Reply:
> The exit link in the header?

No I meant the one in the footer. Sorry for the brevity there.